fractal/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "fractal-startup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fractal:start": "fractal start --sync --port 3000",
+    "fractal:build": "fractal build"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
License should be `UNLICENSED` here. We should also add `private: true` to prevent accidental publishing to NPM.